### PR TITLE
Fix for SummernoteInplaceWidget in widgets.py

### PR DESCRIPTION
**Pull request by smclenithan**
SummernoteInplaceWidget does not work if widget context contains __proxy__ values obtained via ugettext_lazy. This quick hack checks the type of the context value and if it's of type __proxy__ forces unicode evaluation so that it can be properly JSON serialized. Without this check the following error this thrown and the Widget will not render.

`TypeError: Object of type '__proxy__' is not JSON serializable`

### CODE REVIEW

File: django_summernote/widgets.py [hunk 1]
```diff
@@ -66,6 +66,13 @@ def template_contexts(self):
         for option in __summernote_options__:
             v = self.attrs.get(option, summernote_config.get(option))
             if v:
+                # Proxy models are not JSON serializable. Values obtained from the Django core via ugettext_lazy will
+                # return a __proxy__ type that is not serializble using the default serializers.
+                # This quick hack checks to see if the value is a proxy type and if so forces evaluation
+                # so that it can be serialized
+                if '.__proxy__' in str(type(v)):
+                    v = str(v)
+                contexts[option] = v
```
Comment (lqez) [inline django_summernote/widgets.py]: Why do we have `contexts[option] = v` twice?

**Comment by smclenithan** [comment]
Can someone explain what the failing codecov checks are complaining about? Most of the added lines are comments.

File: django_summernote/widgets.py (hunk 1 above)
Comment (smclenithan) [inline django_summernote/widgets.py]: That... was a typo. My mistake.

**Comment by lqez** [comment]
@smclenithan Did you add a test for https://codecov.io/gh/summernote/django-summernote/compare/9da2741962b4407d3cf01e365be37614e5d93140...0d94cee489ed34db2cb03df2c2362c22579d85ec/src/django_summernote/widgets.py?before=django_summernote/widgets.py#L73 ? If not, please add a test for it (when it has `proxy`)

**Comment by smclenithan** [comment]
Well I added a test. But I don't know what version of Django this is testing against. It definitely occurs on the latest 1.11.x

**Pull request merged**